test_util/aws.py

```diff
@@ -127,11 +127,36 @@ def wait_for_stack_deletion(self, wait_before_poll_min=3):
         self.wait_for_status_change('DELETE_IN_PROGRESS', 'DELETE_COMPLETE', wait_before_poll_min)
 
     def get_parameter(self, param):
+        """Returns param if in stack parameters, else returns None
```

Comment:
if we want this behavior, than this function needs to work exactly like `.get()` (https://docs.python.org/3/library/stdtypes.html#dict.get). Don't want sort of that API. (Namely: Need to be able to pass the default).

That said, in all of our cases hard failing / raising an exception if not found would probably be better since we're using this not to introspect and find out what shapes things are, but rather to get out mandatory chunks of templates we launch, where we _should_ know the template.